openquakeplatform/openquakeplatform/vulnerability/views.py

```diff
@@ -117,7 +117,7 @@ def list_entries(request, **kwargs):
     if llrs:
         geninfo = geninfo.filter(llrs=llrs)
     if author:
-        geninfo = geninfo.filter(authors__icontains=author)
+        geninfo = geninfo.filter(authors__unaccent=author)
```

Comment:
I believe this feature might be useful in general, so it was worth investigating and creating a custom field that can be reused whenever we want. For instance, I wouldn't be surprised if Cata, after discovering this issue for that special field, would ask us to implement the same for other fields.